internal/kafka/internal/services/kafka.go

```diff
@@ -131,6 +132,15 @@ func NewKafkaService(connectionFactory *db.ConnectionFactory, clusterService Clu
 	}
 }
 
+func (k *kafkaService) ValidateBillingAccount(externalId string, instanceType types.KafkaInstanceType, billingCloudAccountId string, marketplace *string) *errors.ServiceError {
+	quotaService, factoryErr := k.quotaServiceFactory.GetQuotaService(api.QuotaType(k.kafkaConfig.Quota.Type))
```

Comment:
maybe quotaErr would make more sense here (instead of `factoryErr`)?

Reply:
this is calling the quota service factory, not the quota service itself. So I think the name is ok here.